.pre-commit-config.yaml

```
@@ -0,0 +1,26 @@
+repos:
+  -   repo: https://github.com/PyCQA/isort
+      rev: 5.6.4
+      hooks:
+      - id: isort
+  -   repo: https://gitlab.com/pycqa/flake8
+      rev: 3.8.4
+      hooks:
+      - id: flake8
+  -   repo: https://github.com/pre-commit/pre-commit-hooks
+      rev: v3.3.0
+      hooks:
+      -   id: trailing-whitespace
+      -   id: end-of-file-fixer
+      -   id: check-case-conflict
+      -   id: check-json
+  -   repo: https://github.com/pre-commit/mirrors-eslint
+      rev: '7.15.0'
```

This repo prefixes versions with "v".

```suggestion
      rev: v7.15.0
```